src/main/c/netty_quic_quiche.c

```diff
@@ -237,6 +237,26 @@ static jboolean netty_quic_quiche_conn_is_closed(JNIEnv* env, jclass clazz, jlon
     return quiche_conn_is_closed((quiche_conn *) conn) == true ? JNI_TRUE : JNI_FALSE;
 }
 
+static jlongArray netty_quic_quiche_conn_stats(JNIEnv* env, jclass clazz, jlong conn) {
+    quiche_stats stats;
+    quiche_conn_stats((quiche_conn *) conn, &stats);
```

Comment:
Not sure if this can ever happen but better safe then sorry 
```suggestion
    quiche_conn_stats((quiche_conn *) conn, &stats);
    if (stats == NULL) {
        return NULL;
    }
```